programs/multisig/src/lib.rs

```diff
@@ -188,53 +188,50 @@ pub mod serum_multisig {
 #[derive(Accounts)]
 pub struct CreateMultisig<'info> {
     #[account(zero, signer)]
-    multisig: ProgramAccount<'info, Multisig>,
+    multisig: Box<Account<'info, Multisig>>,
 }
 
 #[derive(Accounts)]
 pub struct CreateTransaction<'info> {
-    multisig: ProgramAccount<'info, Multisig>,
+    multisig: Box<Account<'info, Multisig>>,
     #[account(zero)]
-    transaction: ProgramAccount<'info, Transaction>,
+    transaction: Box<Account<'info, Transaction>>,
     // One of the owners. Checked in the handler.
-    #[account(signer)]
-    proposer: AccountInfo<'info>,
+    proposer: Signer<'info>,
 }
 
 #[derive(Accounts)]
 pub struct Approve<'info> {
     #[account(constraint = multisig.owner_set_seqno == transaction.owner_set_seqno)]
-    multisig: ProgramAccount<'info, Multisig>,
+    multisig: Box<Account<'info, Multisig>>,
     #[account(mut, has_one = multisig)]
-    transaction: ProgramAccount<'info, Transaction>,
+    transaction: Box<Account<'info, Transaction>>,
     // One of the multisig owners. Checked in the handler.
-    #[account(signer)]
-    owner: AccountInfo<'info>,
+    owner: Signer<'info>,
 }
 
 #[derive(Accounts)]
 pub struct Auth<'info> {
     #[account(mut)]
-    multisig: ProgramAccount<'info, Multisig>,
+    multisig: Box<Account<'info, Multisig>>,
```

Comment:
note: it's not required to use `Box<...>` all time because a small struct should be able to fit into bpf stack size.